examples/network/edgeStyles/selfReference.html

```diff
@@ -26,100 +26,30 @@
             {
                 id: 1,
                 label:
-                    "I'm an alligator,\nI'm a mama-papa coming for you.\nI'm the space invader,\nI'll be a rock 'n' rollin' bitch for you",
+                    "Node has selfReference edge,\nwhich angle (position) is provided.\nProperty renderBehindTheNode allows to draw parts,\nwhich are not overlaping with the node",
```

Comment:
A code readability tip here: You can use the following pattern instead of having a really long string with arbitrarily placed `\n`s:
```javascript
const multilineString = [
  "The first line goes here.",
  "The second line comes right bellow it.",
  "Here goes the third.",
  "Finally the fourth and last line."
].join("\n");
```